CHANGELOG.md

```diff
@@ -208,17 +208,38 @@ Other minor additions
 
 * Added new proofs to `Data.Nat.Properties`:
   ```agda
-  even≢odd     : ∀ m n → 2 * m ≢ suc (2 * n)
-  0≢1+n        : 0 ≢ suc n
-  n<1+n        : n < suc n
-  0<1+n        : 0 < suc n
-  m<m+n        : n > 0 → m < m + n
-  m≤n⇒m<n∨m≡n  : m ≤ n → m < n ⊎ m ≡ n
+  0≢1+n          : 0 ≢ suc n
+  1+n≢n          : suc n ≢ n
+  even≢odd       : 2 * m ≢ suc (2 * n)
+
+  0<1+n          : 0 < suc n
+  n<1+n          : n < suc n
+  m<m+n          : n > 0 → m < m + n
+  m<n⇒n≢0        : m < n → n ≢ 0
+  m<n⇒m≤1+n      : m < n → m ≤ suc n
+  m≤n⇒m<n∨m≡n    : m ≤ n → m < n ⊎ m ≡ n
+  ∀[m≤n:m≢o]⇒o<n : (∀ {m} → m ≤ n → m ≢ o) → n < o
+  ∀[m<n:m≢o]⇒o≤n : (∀ {m} → m < n → m ≢ o) → n ≤ o
 
```

Comment:
Should these `:` be `⇒` instead?

Reply:
Hmm I'm not sure. I was trying to mimic universal quantification syntax, i.e. `∀ x : P(x)`. Don't feel strongly about it so I guess I'll change it :+1: